plugins/remsql/fdb_comm.c

```diff
@@ -1519,8 +1571,6 @@ static const char *__tran_2_str(enum transaction_level lvl)
         return "SOSQL";
     if (lvl == TRANLEVEL_RECOM)
         return "RECOM";
-    if (lvl == TRANLEVEL_MODSNAP)
-        return "MODSNAP";
```

Comment:
Did you mean to delete this?

Reply:
no, it was bad merge of recent changes. Thanks